From 8f31a5f68df1d3be7813ddc81359c6a668e78f49 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 28 Feb 2019 23:56:35 +0100
Subject: [PATCH v2 1/1] efi_loader: HII protocols: informative error message

When package types are not supported an error message is displayed.
Unfortunately there is no hint in the messages that they concern the HII
protocols. Furthermore they are indent by tab for no obvious reason.

Prefix the messages by "EFI HII: " instead of "\t".

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2
	Use "EFI HII:" as prefix
---
 lib/efi_loader/efi_hii.c | 36 ++++++++++++++++++------------------
 1 file changed, 18 insertions(+), 18 deletions(-)

diff --git a/lib/efi_loader/efi_hii.c b/lib/efi_loader/efi_hii.c
index 0ed4b196333..38e54969c84 100644
--- a/lib/efi_loader/efi_hii.c
+++ b/lib/efi_loader/efi_hii.c
@@ -388,7 +388,7 @@ add_packages(struct efi_hii_packagelist *hii,
 				(struct efi_hii_guid_package *)package);
 			break;
 		case EFI_HII_PACKAGE_FORMS:
-			printf("\tForm package not supported\n");
+			printf"EFI HII: Form package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_STRINGS:
@@ -396,19 +396,19 @@ add_packages(struct efi_hii_packagelist *hii,
 				(struct efi_hii_strings_package *)package);
 			break;
 		case EFI_HII_PACKAGE_FONTS:
-			printf("\tFont package not supported\n");
+			printf"EFI HII: Font package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_IMAGES:
-			printf("\tImage package not supported\n");
+			printf"EFI HII: Image package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_SIMPLE_FONTS:
-			printf("\tSimple font package not supported\n");
+			printf"EFI HII: Simple font package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_DEVICE_PATH:
-			printf("\tDevice path package not supported\n");
+			printf"EFI HII: Device path package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_KEYBOARD_LAYOUT:
@@ -416,7 +416,7 @@ add_packages(struct efi_hii_packagelist *hii,
 				(struct efi_hii_keyboard_package *)package);
 			break;
 		case EFI_HII_PACKAGE_ANIMATIONS:
-			printf("\tAnimation package not supported\n");
+			printf"EFI HII: Animation package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_END:
@@ -522,33 +522,33 @@ update_package_list(const struct efi_hii_database_protocol *this,
 			remove_guid_package(hii);
 			break;
 		case EFI_HII_PACKAGE_FORMS:
-			printf("\tForm package not supported\n");
+			printf"EFI HII: Form package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_STRINGS:
 			remove_strings_package(hii);
 			break;
 		case EFI_HII_PACKAGE_FONTS:
-			printf("\tFont package not supported\n");
+			printf"EFI HII: Font package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_IMAGES:
-			printf("\tImage package not supported\n");
+			printf"EFI HII: Image package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_SIMPLE_FONTS:
-			printf("\tSimple font package not supported\n");
+			printf"EFI HII: Simple font package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_DEVICE_PATH:
-			printf("\tDevice path package not supported\n");
+			printf"EFI HII: Device path package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_KEYBOARD_LAYOUT:
 			remove_keyboard_package(hii);
 			break;
 		case EFI_HII_PACKAGE_ANIMATIONS:
-			printf("\tAnimation package not supported\n");
+			printf"EFI HII: Animation package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			break;
 		case EFI_HII_PACKAGE_END:
@@ -609,7 +609,7 @@ list_package_lists(const struct efi_hii_database_protocol *this,
 				break;
 			continue;
 		case EFI_HII_PACKAGE_FORMS:
-			printf("\tForm package not supported\n");
+			printf"EFI HII: Form package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			continue;
 		case EFI_HII_PACKAGE_STRINGS:
@@ -617,19 +617,19 @@ list_package_lists(const struct efi_hii_database_protocol *this,
 				break;
 			continue;
 		case EFI_HII_PACKAGE_FONTS:
-			printf("\tFont package not supported\n");
+			printf"EFI HII: Font package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			continue;
 		case EFI_HII_PACKAGE_IMAGES:
-			printf("\tImage package not supported\n");
+			printf"EFI HII: Image package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			continue;
 		case EFI_HII_PACKAGE_SIMPLE_FONTS:
-			printf("\tSimple font package not supported\n");
+			printf"EFI HII: Simple font package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			continue;
 		case EFI_HII_PACKAGE_DEVICE_PATH:
-			printf("\tDevice path package not supported\n");
+			printf"EFI HII: Device path package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			continue;
 		case EFI_HII_PACKAGE_KEYBOARD_LAYOUT:
@@ -637,7 +637,7 @@ list_package_lists(const struct efi_hii_database_protocol *this,
 				break;
 			continue;
 		case EFI_HII_PACKAGE_ANIMATIONS:
-			printf("\tAnimation package not supported\n");
+			printf"EFI HII: Animation package not supported\n");
 			ret = EFI_INVALID_PARAMETER;
 			continue;
 		case EFI_HII_PACKAGE_END:
-- 
2.20.1

